File: POO PHP/Aula12/Mamifero.php
<?php

require_once 'Animal.php';

class Mamifero extends Animal{

    //Atributo da Classe
    private $corPelo;
    
    //Metodo Acessor e Modificador
    public function getCorPelo() {
        return $this->corPelo;
    }
    public function setCorPelo($corPelo): void {
        $this->corPelo = $corPelo;
    }

    //Metodos Sobreposto da Classe Animal
    public function alimentar() {
        echo "<p>Comendo coisas de mamifero ...</p>";
    }

    public function emitirSom() {
        echo "<p>Emitindo som de mamifero ....</p>";
    }

    public function locomover() {
        echo "<p>Se locomovendo como mamifero ....</p>";
    }
}

File: POO JAVA/Aula13/src/aula13/Mamifero.java
package aula13;

public class Mamifero extends Animal{

    //Atributo da Classe
    protected String corPelo;
    
    //Metodo Acesso e Modificador
    public String getCorPelo() {
        return corPelo;
    }
    public void setCorPelo(String corPelo) {
        this.corPelo = corPelo;
    }
    
    //Metodo de Sobreporsição da Classe Animal
    @Override
    public void emitirSom() {
        System.out.println("Emitindo som de mamifero ...");
    }
    
}
